src/test/resources/field/registerFieldsObject.json

```diff
@@ -1,12 +1,23 @@
 {
   "indexName": "test_index",
   "field": [
+    {
+      "name": "real_id",
+      "type": "_ID",
+      "search": true,
+      "store": true
+    },
     {
       "name": "doc_id",
       "type": "ATOM",
       "search": true,
       "storeDocValues": true
     },
+    {
```

Comment:
currently I require the user to add these 2 fields in the schema explicitly. I can hide this to the user. Wondering which approach is preferred. I feel some query like empty query may end up return child docs. The user can use _nested_type to indicates they only want parent docs